jujugui/static/gui/src/app/components/scale-service/_scale-service.scss

```diff
@@ -0,0 +1,51 @@
+.scale-service {
+
+    input[type=text] {
```

Comment:
Yes that should probably be set app wide so I won't tackle it here.

done converting to class.